addons/binding/org.openhab.binding.chromecast/src/main/java/org/openhab/binding/chromecast/handler/ChromecastHandler.java

```diff
@@ -262,6 +266,21 @@ public void spontaneousEventReceived(final ChromeCastSpontaneousEvent event) {
         }
     }
 
+    String getMimeType(String mediaUrl) {
+        URL url = null;
+        String mimeType = null;
+        try {
+            url = new URL(mediaUrl);
+            HttpURLConnection connection = (HttpURLConnection) url.openConnection();
+            connection.connect();
+            mimeType = connection.getContentType();
+            connection.disconnect();
+        } catch (IOException e) {
+            // we couldn't establish mime type
```

Comment:
We should at least be logging something here, rather than just swallowing the error.